intro.typ

```diff
@@ -1 +1,5 @@
 = Introduction
+== NASA Mission Statement
+NASA strives to explore the unknown throughout the atmosphere and in space. As a company, it is NASA's mission to benefit humanity through innovation, and inspire through discovery. NASA fosters a curious, resilient and passionate community to lead to the development of innovative solutions for all things air and space. When focusing specifically on Mars Exploration, NASA claims that the mission for Mars exploration is to "understand[ing] the past, present or future potential for life on Mars." There are 4 different overarching goals for NASA Mars Exploration, and the one the Mars C3 system contributes to is Preparing for Human Exploration. Part of traveling safely to Mars and surviving an environment as hostile as Mars' is having a reliable and fast communication system to get instructions and get help if an emergency arises. 
```

Comment:
> NASA claims that the mission for Mars exploration is to "understand[ing] the past, present or future potential for life on Mars."

Should probably have attribution.

- Consider using a quote: https://typst.app/docs/reference/model/quote/
- and/or citing a reference